packages/comet-admin/src/mui/MasterLayout.tsx

```diff
@@ -4,18 +4,19 @@ import { createStyles, WithStyles, withStyles } from "@material-ui/styles";
 import * as React from "react";
 
 import { Header } from "./MasterLayout.sc";
-import { MenuContext } from "./menu/Context";
+import { MenuContext } from "./menu";
 
 export interface IMasterLayoutProps extends WithStyles<typeof styles> {
     children: React.ReactNode;
     menuComponent: React.ComponentType;
     headerComponent?: React.ComponentType;
     hideToolbarMenuIcon?: boolean;
+    closeMenuInitially?: boolean;
 }
 
 function MasterLayout(props: IMasterLayoutProps) {
-    const { classes, children, menuComponent: Menu, headerComponent: HeaderComponent, hideToolbarMenuIcon } = props;
-    const [open, setOpen] = React.useState(true);
+    const { classes, children, menuComponent: Menu, headerComponent: HeaderComponent, hideToolbarMenuIcon, closeMenuInitially = false } = props;
```

Comment:
I prefer `openByDefault = true` instead of `closeMenuInitially = false `
Would be consistent then with in prop-name in CollapsibleItem 
https://github.com/vivid-planet/comet-admin/pull/280/files#diff-ea3b3c7dc8d47177460310060c8f44eae005af54ff49d39dae8a9de34283aadcR32